projects/cdk/decorators/required-setter.ts

```diff
@@ -15,21 +15,25 @@ export function tuiRequiredSetter<T extends object, K extends keyof T>(
         target: Object,
         key,
         {configurable, enumerable, get, set}: PropertyDescriptor,
-    ) => {
+    ): PropertyDescriptor => {
         const {name} = target.constructor;
 
-        Object.defineProperty(target, key, {
+        return {
             configurable,
             enumerable,
             get,
-            set(this: T, value: T[K]) {
+            set(this: T, value: T[K]): void {
```

Comment:
We do not write `: void` as per our code style

Reply:
Ok then! Removed.